pytket/tests/requirements.txt

```diff
@@ -7,6 +7,6 @@ py
 hypothesis
 docker
 jsonschema
-quimb
+quimb;python_version<"3.11"
```

Comment:
Do you know if they have any plans for 3.11 support in the near future?

Reply:
I would expect so, since it seems to be quite well maintained.